cmd/users.go

```diff
@@ -366,10 +366,10 @@ var getAllUserKeysCmd = &cobra.Command{
 	},
 }
 
-var addUserToOrganizationCmd = &cobra.Command{
-	Use:     "user",
-	Aliases: []string{"u"},
-	Short:   "Add a user to an Organization",
+var addAdministratorToOrganizationCmd = &cobra.Command{
+	Use:     "organization-administrator",
+	Aliases: []string{"org-admin"},
+	Short:   "Add an administrator to an Organization",
```

Comment:
Might be worth extending this just to include the note about the default being viewers, something like this
```
Long: `Add an administrator to an Organization
If the owner flag is not provided users will be added as viewers`
```